pkg/operator/startupmonitorreadiness/readiness_checks.go

```diff
@@ -377,3 +381,22 @@ func filterByNodeName(kasPods []corev1.Pod, currentNodeName string) []corev1.Pod
 
 	return filteredKasPods
 }
+
+func getExcludedChecks() []string {
+	return []string{
+		// Speed up SNO install by declaring kube-apiserver healthy before openshift-apiserver is ready
```

Comment:
should we also mention that this is SAFE because the location of the openshift-apiserver is hardcoded in the 
kube-apiserver and it is not part of the configuration/secret?

Reply:
I don't think its useful here - iiuc any bogus value in "except" will be ignored , but I don't mind adding it

Reply:
we could add a comment here and at the source (https://github.com/openshift/kubernetes/blob/53f3066f47d3a594d6b93843916e8711ee673715/openshift-kube-apiserver/openshiftkubeapiserver/sdn_readyz_wait.go)

Reply:
Sure, I'll add the comment there once this merges